src/lib.rs

```diff
@@ -281,7 +281,7 @@ impl DB {
     }
 
     /// Get a [`RawDbIter`] in given range and direction.
-    pub(crate) fn raw_iter_range<S: Schema>(
+    pub fn raw_iter_range<S: Schema>(
```

Comment:
Why this needs to be public?

Reply:
It's temp for testing, don't review this for now please :smile:

Reply:
Oh, ok!